### PR TITLE
Added a "restart packager" command.

### PR DESCRIPTION
**Pull request by BretJohnson**
It's in addition to the existing "start packager" and "stop packager" commands. It also resets the cache when (re)starting, which can fix up various issues like source map not being updated for TypeScript apps, etc.

### CODE REVIEW

File: src/common/packager.ts [hunk 1]
```diff
@@ -94,7 +97,7 @@ export class Packager {
             if (running) {
                 if (!this.packagerProcess) {
                     Log.logWarning(ErrorHelper.getWarning("Packager is still running. If the packager was started outside VS Code, please quit the packager process using the task manager."));
-                    return Q.resolve<void>(void 0);
+                    return Q.resolve<void>(void 1);
```
Comment (MSLaguana) [inline src/common/packager.ts]: What's the reason for this change? The expression `void 0` is just a way of producing a "value" of type void; the `void` consumes its argument and doesn't return anything.

Comment (BretJohnson) [inline src/common/packager.ts]: That's a mistake, from me playing around with something. I just fixed it.

File: src/common/error/internalErrorCode.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export enum InternalErrorCode {
         FailedToStartPackager = 106,
         FailedToStopPackager = 107,
         PackagerRunningInDifferentPort = 108,
+        FailedToRestartPackager = 109,
```
Comment (MSLaguana) [inline src/common/error/internalErrorCode.ts]: You will also need to add the string for this error in https://github.com/Microsoft/vscode-react-native/blob/master/errorStrings/errorStrings.json

Comment (BretJohnson) [inline src/common/error/internalErrorCode.ts]: Fixed.

**Comment by MSLaguana** [comment]
Looks good to me.
